tests/jsmarionette/client/marionette-client/lib/marionette/client.js

```diff
@@ -1322,30 +1323,39 @@
      *
      * @method executeScript
      * @chainable
-     * @param {String} script to run.
-     * @param {Array} [args] optional args for script.
-     * @param {Function} callback will receive result of the return \
-     *                            call in the script if there is one.
-     * @param {String} optional sandbox is a tag referring to the sandbox you
-     *                 wish to use; if you specify a new tag, a new sandbox
-     *                 will be created. If you use the special tag 'system',
-     *                 the sandbox will be created using the system principal
-     *                 which has elevated privileges.
-     * @return {Object} self.
+
+     * @param {string} script
+     *     The script to evaluate.
+     * @param {Array.<?>} args
+     *     Optional args for script that will  be available as
+     *     {@code arguments} in the scope the script runs in.
+     * @param {function} callback
+     *     The callback will receive result of the return call in the
+     *     script if there is one.
+     * @param {string} sandbox
+     *     Name of the sandbox to evaluate the script in.  If undefined,
+     *     the default sandbox will be used.  If you specify "system",
+     *     the sandbox will have the system principal which gives
+     *     elevated privileges.
+     *
+     * @return {Marionette}
+     *     Reference to self.
      */
     executeScript: function executeScript(script, args, callback, sandbox) {
-      if (typeof(args) === 'function') {
+      if (typeof args == 'function') {
```

Comment:
This isn’t actually necessary since _typeof_ is guaranteed to always return a string, but I’m happy to add the tests back for consistency.